### ISSUE TITLE
a11y: Paragraph shouldn't be labelled h2

### ISSUE THREAD

**Issue by pearlyong** [opengovsg/FormSG#5501]
**Checked:**
NVDA: Paragraph is reading as h2
Mac Voiceover: the entire heading and paragraph reads together (heading level 2, 2 items)
iPhone Voiceover: Paragraph reads as h2
Talkback: reads out heading and paragraph together. but does not announce h2 

**Proposal:**
Paragraph is only reading out as h2 on NVDA and iPhone Voiceover, could we try and check why?

**Comment by foochifa** [comment]
This happens because the instructions/sectionField has 'role=heading' attached to it.

When the html element has a role=heading, aria will give it a default level of 2, and will read out the entire block as 'heading level 2' in Mac Voiceover.

See [here](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/heading_role)

Have read from some threads that such heading informations are useful for users to navigate the page. So not sure if removing it will be correct, perhaps it makes more sense to add an aria-level to it? So Instruction is Heading 1, section headers are Heading 2 etc

**Comment by foochifa** [comment]
Can't seem to replicate h2 Issue in NVDA, form is correctly reading as 'Instructions ... heading level 2', instead of 'h2'

**Comment by foochifa** [comment]
For iOS and NVDA, as the role heading is given to the entire div, the paragraph will also contain the heading level 2 label. Hence it will read out as instructions ... paragraph ... heading level 2

@staceytan1998 is the correct behaviour:
Instructions Heading Level 2
Paragraph details ... (no heading level 2 here)?
